module/questions/transformationHelper.js

```diff
@@ -0,0 +1,363 @@
+// Dependencies.
+const { isEmpty, find, capitalize } = require("lodash");
+const { readQuestionSet, readQuestion } = require(ROOT_PATH +
+  "/generics/services/knowledge-platform");
+const { questionType } = require(MODULES_BASE_PATH +
+  "/questions/questionTemplate");
+const { criteriaTemplate, defaultCriteria } = require(MODULES_BASE_PATH +
+  "/criteria/criteriaTemplate");
+const { baseAssessment, assessmentTemplate } = require(MODULES_BASE_PATH +
+  "/assessments/assessmentTemplate");
+
+module.exports = class Transformation {
+  static getQuestionSetHierarchy(
+    submissionDocumentCriterias,
+    solutionDocument,
+    isPageQuestionsRequired = true
+  ) {
+    return new Promise(async (resolve, reject) => {
```

Comment:
@swetha-aggidevara Please use try catch and return statements. Refer - https://github.com/shikshalokam/ml-survey-service/blob/master/module/observations/helper.js